indico/modules/events/sessions/templates/management/sessions/sessions_list.html

```diff
@@ -0,0 +1,146 @@
+                    <li><a href="#" id="select-all">{% trans %}All{% endtrans %}</a></li>
+                    <li><a href="#" id="select-none">{% trans %}None{% endtrans %}</a></li>
+                </ul>
+            </div>
+            <div class="group left">
+                <a id="add-new-session" class="i-button icon-plus" data-href="{{ url_for('.add_session', event) }}"
+                   data-title="{% trans %}Add new session{% endtrans %}">
+                    {% trans %}Add new session{% endtrans %}
+                </a>
+                <a id="remove-selected-sessions"
+                   class="i-button icon-remove js-requires-selected-row disabled"
+                   data-title="{% trans %}Remove selected sessions{% endtrans %}"
+                   data-method="POST"
+                   data-href="{{ url_for('.delete_sessions', event) }}"
+                   data-confirm="{% trans %}Do you really want to delete selected session?{% endtrans %}">
+                    {%- trans %}Remove{% endtrans -%}
+                </a>
+            </div>
+            <div class="group left">
+                <a class="i-button arrow button js-requires-selected-row disabled" data-toggle="dropdown">
+                    {%- trans %}Export{% endtrans -%}
+                </a>
+                <ul class="dropdown">
+                    <li>
+                        <a data-href="{{ url_for('.export_pdf', event) }}"
+                           class="icon-file-pdf js-requires-selected-row js-export-sessions disabled">
+                            {%- trans %}PDF{% endtrans -%}
+                        </a>
+                    </li>
+                    <li>
+                        <a data-href="{{ url_for('.export_csv', event) }}"
+                           class="icon-file-excel js-requires-selected-row js-export-sessions disabled">CSV</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+            {% if sessions -%}
+                <table class="i-table tablesorter">
+                    <thead>
+                        <tr class="i-table">
+                            <th class="i-table checkbox-column" data-sorter="false"></th>
+                            <th class="i-table id-column">{% trans %}ID{% endtrans %}</th>
+                            <th class="i-table" data-sorter="false">
+                                {%- trans %}Blocks{% endtrans -%}
+                            </th>
+                            <th class="i-table">{% trans %}Title{% endtrans %}</th>
+                            <th class="i-table">{% trans %}Type{% endtrans %}</th>
+                            <th class="i-table" data-sorter="false">{% trans %}Colour{% endtrans %}</th>
+                            <th class="i-table" data-sorter="false"></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for session in sessions -%}
```

Comment:
i'd put the whole loop / tbody content in a macro so you can render it server-side and send it back after modifying things (could even be a macro for each row so you just send the modified row back instead of everything)